README.md

```diff
@@ -1,4 +1,15 @@
 # ThavalonWeb
-Web App for THavalon
+This repo contains the front-end web application, back-end API, and database code for ThavalonWeb. For a description of Thavalon, see below. To view the rules, see the [rules page](docs/rules.md).
```

Comment:
The front end + back end + database is everything, right?

Reply:
Changed